README.md

```diff
@@ -12,6 +12,7 @@ This repository contains examples of [embedding Observable notebooks](https://ob
 | [**custom-fluid-width**](https://github.com/observablehq/examples/tree/main/custom-fluid-width/) | Resize a chart when the window is resized |
 | [**custom-fluid-width-and-height**](https://github.com/observablehq/examples/tree/main/custom-fluid-width-and-height/) | Resize a chart when its container is resized |
 | [**custom-library**](https://github.com/observablehq/examples/tree/main/custom-library/) | Override the Observable Standard Library |
+| [**secret**](https://github.com/observablehq/examples/tree/main/secret) | Override the implementation of Secrets |
```

Comment:
```suggestion
| [**secret**](https://github.com/observablehq/examples/tree/main/secret) | Override the implementation of secrets |
```

It’s good to capitalize the `Secret` function, but I’d prefer to keep the term “secrets” lowercase.